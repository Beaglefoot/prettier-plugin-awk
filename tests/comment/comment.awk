{
    x=a+b # Just random comment

    # prettier-ignore
    x=a+b
}
