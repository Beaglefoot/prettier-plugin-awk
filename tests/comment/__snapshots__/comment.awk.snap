{
    x=a+b # Just random comment

    # prettier-ignore
    x=a+b
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    x = a + b # Just random comment

    # prettier-ignore
    x=a+b
}
